client/components/common/tce-core/ElementList.vue

```diff
@@ -69,9 +69,7 @@ export default {
     ...mapChannels({ editorChannel: 'editor' }),
     options: vm => ({
       ...vm.dragOptions,
-      handle: '.drag-handle',
```

Comment:
According to this => https://www.npmjs.com/package/vuedraggable#all-sortable-options
This is the current list of sortable options =>  https://github.com/SortableJS/sortablejs/blob/master/docs/sortable-options.md and these props are removed so 👍

Reply:
It is not included by default => https://github.com/SortableJS/Vue.Draggable/blob/master/src/vuedraggable.js